### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by michaelbowler**
## PR summary
ID edits for README as per https://github.ibm.com/org-ids/ua-issues/issues/1939

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [x ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by michaelbowler** [comment]
@gmarjoram @cbrealey - could we reviewmerge this PR if OK?

**Comment by gmarjoram** [comment]
@michaelbowler is the table label meant to be under the table ?
image

**Comment by michaelbowler** [comment]
@gmarjoram - hum, yes. Let me amend...

**Comment by michaelbowler** [comment]
@gmarjoram ... better:

image

**Review by gmarjoram — APPROVED** [review]
I'm ok with this but would like @cbrealey to also have a quick look

**Comment by michaelbowler** [comment]
@briangleeson - not sure of your process here.  Can I resolve conflicts and merge now?

**Comment by briangleeson** [comment]
@michaelbowler you can go ahead and resolve the merge conflicts. From reading above, I think we're just waiting for approval from @cbrealey on the changes, then can merge

**Review by briangleeson — CHANGES_REQUESTED** [review]

File: README.md
Comment (michaelbowler) [reply README.md:63]: OK NP @briangleeson

**Review by briangleeson — APPROVED** [review]

**Review by cbrealey — APPROVED** [review]
Changes look excellent, and apologies for the delay on my part. Only side comment is that this README may be originally been based on a template, in which event we should seek out and update that template so that other IBM SDK repos' READMEs may benefit.

**Comment by michaelbowler** [comment]
Thanks @cbrealey - can we now close https://github.ibm.com/org-ids/ua-issues/issues/1939 also?

I'll also hunt for and change the source template.

**Pull request merged**
